README.md

```diff
@@ -38,6 +38,9 @@ See the [Install README](/docs/README.md) for more details on the installation.
 
 See the [API README](/docs/API.md) for more details on the Kruize REST API.
 
+## Additional runtimes recommendations
+
+See the [Runtimes README](/docs/Runtimes.md) for more details on how to generate additional runtimes specific recommendations for your application through Kruize.
```

Comment:
Can we add a bit more detail on the current recommendation that we provide. Add a section on `Heap size recommendation` and add more details about what exactly can the user expect here. We also need to add the current runtimes supported (Only Java, and only OpenJ9 for now).